sinks/prometheus/prometheus.go

```diff
@@ -0,0 +1,143 @@
+package prometheus
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+	"text/template"
+
+	"github.com/stripe/veneur/samplers"
+	"github.com/stripe/veneur/sinks"
+	"github.com/stripe/veneur/ssf"
+	"github.com/stripe/veneur/trace"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// Max package size for TCP (and hardcoded for UDP in Statsd Exporter) is
+	// 65535 bytes. Assuming a generous 50 bytes for metric name, 10 bytes for
+	// value, and 200 bytes for tags, we can have ~200 packets per batch.
+	batchSize = 200
+
+	// Use DogstatsD serialization format.
+	// https://github.com/prometheus/statsd_exporter#tagging-extensions.
+	serializationFormat = "{{.Name}}:{{.Value}}|{{.Type}}|#{{.Tags}}\n"
```

Comment:
Likewise, we probably want this to be a const

Reply:
This is done preemptively  for the same reasons, though probably shouldn't since it doesn't make tests cleaner, will change this